src/lib/serviceRoutines/getContextEntitiesByEntityId.cpp

```diff
@@ -49,15 +64,23 @@ std::string getContextEntitiesByEntityId
   ParseData*                 parseDataP
 )
 {
-  std::string                          entityId = (compV[0] == "v1")? compV[3] : compV[2];
-  std::string                          answer;
-  DiscoverContextAvailabilityResponse  response;
+  std::string  entityId = (compV[0] == "v1")? compV[3] : compV[2];
+  std::string  answer;
+
+  //
+  // Fill in parseDataP->dcar.res to pass to postDiscoverContextAvailability
+  //
+  EntityId                             eId(entityId, "", "");
+  std::vector<std::string>             attributeV;
+  Restriction                          restriction;
+
+  parseDataP->dcar.res.fill(eId, attributeV, restriction);
```

Comment:
This fill method doesn't know where it is used. Could be from a convop or from anywhere else.
I think 'fill' is an excellent name :-)

NTC